server/backend/users/views.py

```
@@ -57,7 +57,7 @@ def post(self, request):
             token.blacklist()  # Blacklist the token
 
             return Response({"message": "Logout successful"}, status=status.HTTP_205_RESET_CONTENT)
-        except Exception as e:
+        except Exception as _:
```

Ruff (0.8.2)

Do not catch blind exception: `Exception`

(BLE001)